drivers/bluetooth/hci/hci_nxp.c

```diff
@@ -0,0 +1,1178 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <stddef.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/arch/cpu.h>
+
+#include <zephyr/init.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/sys/byteorder.h>
+#include <string.h>
+
+#define LOG_LEVEL CONFIG_BT_HCI_DRIVER_LOG_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_nxp_ctlr);
+
+#include "common/bt_str.h"
+
+#include "uart_nw61x_se.h"
+
+static const struct device *uart_dev = DEVICE_DT_GET(DT_CHOSEN(zephyr_bt_uart));
+
+#define HCI_UART_CHILD_PREFIX(x) x
+#define HCI_UART_CHILD           DT_FOREACH_CHILD(DT_CHOSEN(zephyr_bt_uart), HCI_UART_CHILD_PREFIX)
+#if DT_NODE_HAS_PROP(HCI_UART_CHILD, sdio_reset_gpios)
+struct gpio_dt_spec sido_reset = GPIO_DT_SPEC_GET(HCI_UART_CHILD, sdio_reset_gpios);
```

Comment:
```suggestion
struct gpio_dt_spec sdio_reset = GPIO_DT_SPEC_GET(HCI_UART_CHILD, sdio_reset_gpios);
```
Is this a misspelling? If yes also code below needs to be fixed

Reply:
fixed